tests/admin/views/test_releases_v2.py

```diff
@@ -0,0 +1,1173 @@
+                        "complete": False,
+                    },
+                    {
+                        "name": "Firefox-67.0-build1",
+                        "path": ".platforms.Linux_x86-gcc3.locales.en-US",
+                        "data_version": 1,
+                        "sc_id": 13,
+                        "scheduled_by": "bob",
+                        "when": 2222222222000,
+                        "change_type": "delete",
+                        "sc_data_version": 1,
+                        "complete": False,
+                    },
+                    {
+                        "name": "Firefox-67.0-build1",
+                        "path": ".platforms.Linux_x86_64-gcc3.locales.en-US",
+                        "data_version": 1,
+                        "sc_id": 14,
+                        "scheduled_by": "bob",
+                        "when": 2222222222000,
+                        "change_type": "delete",
+                        "sc_data_version": 1,
+                        "complete": False,
+                    },
+                    {
+                        "name": "Firefox-67.0-build1",
+                        "path": ".platforms.WINNT_aarch64-msvc-aarch64.locales.en-US",
+                        "data_version": 1,
+                        "sc_id": 15,
+                        "scheduled_by": "bob",
+                        "when": 2222222222000,
+                        "change_type": "delete",
+                        "sc_data_version": 1,
+                        "complete": False,
+                    },
+                    {
+                        "name": "Firefox-67.0-build1",
+                        "path": ".platforms.WINNT_x86-msvc.locales.en-US",
+                        "data_version": 1,
+                        "sc_id": 16,
+                        "scheduled_by": "bob",
+                        "when": 2222222222000,
+                        "change_type": "delete",
+                        "sc_data_version": 1,
+                        "complete": False,
+                    },
+                    {
+                        "name": "Firefox-67.0-build1",
+                        "path": ".platforms.WINNT_x86_64-msvc.locales.en-US",
+                        "data_version": 1,
+                        "sc_id": 17,
+                        "scheduled_by": "bob",
+                        "when": 2222222222000,
+                        "change_type": "delete",
+                        "sc_data_version": 1,
+                        "complete": False,
+                    },
+                ],
+            },
+            {
+                "name": "CDM-17",
+                "product": "CDM",
+                "data_version": 1,
+                "read_only": False,
+                "rule_info": {"3": {"product": None, "channel": "beta"}},
+                "scheduled_changes": [],
+            },
+            {
+                "name": "Firefox-64.0-build1",
```

Comment:
I can do this, but it actually requires changing the impementation a bit (to sort the Releases before returning them rather than place them in the list in the order we find them). This is probably a sensible thing to do anyways.

Reply:
Oh well, if it's straight forward I guess. Not going to fight for this hill.